tests/AppTest.php

```diff
@@ -30,6 +34,19 @@ public function testLoginWithExistentUserButWrongPassword()
 
     public function testLoginWithExistentUserAndCorrectPassword()
     {
+        // Set Up
+         User::create(
```

Comment:
If you will be creating the same user for all your tests, you can still use the `public function setUp()` method to create it. This method will be executed before each test is run